conf/ocsci/aws_region_us-east-1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+ENV_DATA:
+  platform: 'aws'
```

Comment:
In the [default_config.yaml](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L80) and also in [marks.py](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/marks.py#L62:L65), the platform value is upper-case `AWS` so even though in other config files it is used with lower-case, it might be better/safer to set it upper-case here.

Reply:
Since issue https://github.com/red-hat-storage/ocs-ci/issues/926 was fixed (PR https://github.com/red-hat-storage/ocs-ci/pull/927 is merged), we can probably ignore this note.